app/views/pages/create_good_forms.html.erb

```diff
@@ -0,0 +1,77 @@
+
+<h2 class="govuk-heading-l">Making sure a form is accessible</h2>
+
+<p>
+It’s important that government forms are accessible to everyone, including disabled people using assistive technologies.
+</p>
+
+<p>
+When you create a form with GOV.UK Forms, you’re using designs which have been tested to make sure they meet government accessibility standards. But it’s still possible to create accessibility problems if you use GOV.UK Forms incorrectly.
+</p>
+
+<h3 class="govuk-heading-m">Asking questions in an accessible way</h3>
+
+<p>
+When you create a form, you’ll be asked what kind of answer you need for each question. Make sure you choose the right option: that way the browser knows what kind of answer to expect, and can help the user by filling in parts of the form automatically.
+</p>
+
+<p>
+Avoid asking for the same information more than once within a form.
+</p>
+
+<p>
+Use language that everyone who needs to complete the form will understand. Explain anything that’s not obvious — for example, by adding hint text or guidance to a question.
+</p>
+
+<h3 class="govuk-heading-m">Accessible links and headings</h3>
+
+<p>
+Use link text which tells the user what will happen if they follow the link, even if they’re reading it out of context. For example — ‘Find out more about GOV.UK Forms’. Avoid link text that does not make sense out of context — like ‘Click here’ or ‘Read more’.
+</p>
+
+<p>
+Disabled users often rely on headings to help them navigate around a website page. So if you add a heading to a page in your form, make sure the heading reflects what’s underneath it. For example — to break up text that’s aimed at two different audiences, create two headings. Then put the text that’s aimed at each audience under the relevant heading — avoid ‘orphaning’ any text so it falls under the wrong heading.
+</p>
+
+<h2 class="govuk-heading-l">Linking to a form from the GOV.UK website</h2>
+
+<p>
+You’ll need a link to your form from the GOV.UK website so that people can find it.
+</p>
+
+<p>
+The GOV.UK publishing team in your organisation will be able to advise on whether GOV.UK Forms is the best way to meet your needs — and how to link to the form. Talk to them about your requirements as soon as possible.
+</p>
+
+<h3 class="govuk-heading-m">If you’re in a GOV.UK publishing team</h3>
+
+<p>
+It’s possible to add a link from an existing ‘publication type:form’ page, but where possible we recommend linking to the form from a service-style starting point.
+</p>
+
+<p>
+You can create a starting point using the ‘detailed guide’ format in Whitehall. <a class="govuk-link" href="https://www.gov.uk/guidance/close-your-mot-centre/">DVSA’s ‘Close your MOT centre’ page</a> is an example.
+</p>
+
+<p>
+Use Markdown to create a green ‘Start now’ button:
+</p>
+
+<pre><code>{button start}[Button text goes here](https://submit.forms.service.gov.uk/form/first-page-of-form){/button}</code></pre>
```

Comment:
The Design System website adds `tabindex=0` and some CSS for the `:focused` state to code tags to make sure that they can be focused and scrolled with a keyboard [[1], [2]], we should probably do the same thing.

[1]: https://github.com/alphagov/govuk-design-system/blob/2fc7ffa74ae71c6d869819732f37bedd068fcb67/lib/marked/renderer.js#L13-L15
[2]: https://github.com/alphagov/govuk-design-system/commit/b860e159eb4a38ed300f009b4a58c47ba9fc3e59